themes/custom-theme/theme/base/login/messages/messages_en.properties

```diff
@@ -1,5 +1,7 @@
-doLogIn=Sign In
-doRegister=Register
+doLogIn=Log in
+doRegister=Sign up
+doRegisterHere=Sign up here
+doSubmitResetPassword=Send Password Reset Link
```

Comment:
Shall we adjust the message to `Reset Password`? The longer text feels a little bit out of place on a button

Reply:
Good point, but we don't have any instructions on what to expect when the link will be clicked. I will ask Luuk what would be his suggestion

Reply:
Discussed with Luuk and he came up with a copy text to provide some context 
<img width="673" alt="image" src="https://github.com/user-attachments/assets/2388fdfa-e48b-4392-994f-99d3805e3717">